File: Models/Employee.cs
using System;
using System.Collections.Generic;

namespace AWC.ActivityPortal.Models
{
    public class Employee : EntityBase
    {
        public int ActivityId { get; set; }

        public string IdentityId { get; set; }
        public AppUser Identity { get; set; }
        public Activity Activity { get; set; }
        public ICollection<Comment> Comments { get; set; }

    }
}

File: src/models/employee.ts
import { Type } from 'class-transformer';
import { Activity } from './activity';
import { AppUser } from './app-user';
import { EntityBase } from './entity-base';

export class Employee extends EntityBase {
  activityId: number;
  @Type(() => AppUser)
  identity: AppUser;
  @Type(() => Activity)
  activity: Activity;
  @Type(() => Comment)
  comments: Comment[];
}
